pkg/cmd/server/start/start_master.go

```diff
@@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
 		kc.RunGCController(gcClient)
 
+		kc.RunServiceLoadBalancerController()
```

Comment:
I meant one service account per controller.

On Tue, Apr 26, 2016 at 12:35 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_master.go
> <https://github.com/openshift/origin/pull/8633#discussion_r61119394>:
>
> > @@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
> >  		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
> >  		kc.RunGCController(gcClient)
> >
> > +		kc.RunServiceLoadBalancerController()
>
> One role per thing with discrete and predictable permissions to change API
> objects across all namespaces? That seems pretty reasonable to me.
> … <#m_-8016314581085398451_>
> On Tue, Apr 26, 2016 at 11:33 AM, Clayton Coleman ***@***.***> wrote: In
> pkg/cmd/server/start/start_master.go <#8633 (comment)
> <https://github.com/openshift/origin/pull/8633#discussion_r61108219>>: >
> @@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc
> *kubernetes.MasterConfig) erro >
> kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"),
> recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace) >
> kc.RunGCController(gcClient) > > + kc.RunServiceLoadBalancerController()
> One role per controller? Yuck … <#m_-4805712814360026323_> On Tue, Apr 26,
> 2016 at 11:31 AM, Clayton Coleman ***@***.***> wrote: > Have we taken into
> account how we'll change this code path to use > versioned clients? > > On
> Tue, Apr 26, 2016 at 11:04 AM, David Eads ***@***.***> > wrote: > In
> pkg/cmd/server/start/start_master.go > <#8633
> <https://github.com/openshift/origin/pull/8633> (comment) <#8633 (comment)
> <https://github.com/openshift/origin/pull/8633#discussion_r61102578>>>: >
> > > @@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc
> *kubernetes.MasterConfig) erro > >
> kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"),
> recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace) >
> > kc.RunGCController(gcClient) > > > > +
> kc.RunServiceLoadBalancerController() > > Yeah, there's still a mix of the
> old and new implementations. Been trying > to catch anything new going in.
> > > Agree. > > As a note for whoever does the rebase, we need to sort out
> what we do > about shared cache ACL-ing. @liggitt
> <https://github.com/liggitt> <https://github.com/liggitt> <
> https://github.com/liggitt> > > — > You are receiving this because you
> authored the thread. > Reply to this email directly or view it on GitHub >
> <
> https://github.com/openshift/origin/pull/8633/files/a7a264ec9a036439b5959ec6c5d2d2234057ed81#r61102578>
> > — You are receiving this because you were mentioned. Reply to this email
> directly or view it on GitHub <
> https://github.com/openshift/origin/pull/8633/files/a7a264ec9a036439b5959ec6c5d2d2234057ed81#r61108219>
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8633/files/a7a264ec9a036439b5959ec6c5d2d2234057ed81#r61119394>
>